doc/content/reference/frequency-plans/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Frequency Plans"
+description: "Frequency plans"
+---
+
+This reference contains Frequency Plans supported by {{% tts %}}. This list is populated from the [lorawan-frequency-plans Github repository.](https://github.com/TheThingsNetwork/lorawan-frequency-plans/blob/master/frequency-plans.yml)
+
+<!--more-->
+
```

Comment:
Maybe note that deployments may have different/custom frequency plans configured, and that these are just the "official" ones.